### DISCUSSION TITLE
Can argo rollout deploy multiple canary versions?

### DISCUSSION THREAD

**Discussion by JiaminZhu** [argoproj/argo-rollouts#3300] [General]
Can Argo Rollout accommodate multiple canary versions simultaneously within the same managed Rollout resource? For instance, having a stable version running alongside two distinct canary versions within the same Rollout resource. Is there a way to achieve this capability in Argo Rollout?

**Comment by zachaller** [comment]
Argo Rollouts Experiments might give you what you want?
